.github/workflows/update-schedule.yml

```diff
@@ -0,0 +1,46 @@
+name: Update schedule.yaml
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * 1'  # Every monday
+jobs:
+  create-pull-request:
+    name: Create PR (if required)
+    if: github.repository == 'kubernetes/website'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-go@v5
+        with:
+          go-version: '1.22'
+          check-latest: true
+
+      - name: Install schedule-builder
+        run: go install k8s.io/release/cmd/schedule-builder@latest
```

Comment:
Maybe pin this rather than using `latest`

Reply:
we should add that to our release-actions, will do that

Reply:
Pinned to the latest version for now.